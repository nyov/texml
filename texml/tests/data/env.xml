<env name="document">
text
<env name="st" begin="s" end="e">ET phone home!</env>
text
</env>
