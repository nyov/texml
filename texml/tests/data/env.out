\begin{document}
text
\s{st}
ET phone home!
\e{st}
text
\end{document}
